torch/csrc/jit/script/ir_emitter.cpp

```diff
@@ -2437,6 +2437,26 @@ struct to_ir {
 
         return std::make_shared<SimpleValue>(expr);
       }
+      case prim::rpc_async: {
+        auto& inputs_trees = apply.inputs().tree()->trees();
+        if (inputs_trees.size() < 2) {
+          throw ErrorReport(apply)
+              << "Expected at least 3 arguments to "
+              << "rpc_async(dst_worker_name, user_func_qual_name, user_func, *args, **kwargs)";
+        }
+        auto dst_worker_name = emitSugaredExpr(Expr(inputs_trees[0]), 1);
+        auto user_func = emitSugaredExpr(Expr(inputs_trees[1]), 1);
+        TreeList args_trees(inputs_trees.begin() + 2, inputs_trees.end());
```

Comment:
if we create node based on rpc_async() signature, here could we parse them differently?

Reply:
Yeah, if args and kwargs could be default to None and users don't pass them, we need to test the input size and do extra handling here.